htmlelements.js

```diff
@@ -0,0 +1,88 @@
+const rootElem = document.getElementById("root");
```

Comment:
Thank you Benyam, for the great feedback. I haven't noticed them at all you are a very focused reviewer. I made all the changes you suggested, only I wanted to keep the number 8 suggestion as it is, because I quite liked to use it in that way. Thank you so much again.